lib/Tool/Session.php

```diff
@@ -29,83 +30,165 @@ final class Session
      */
     private static $handler;
 
-    public static function getHandler(): AdminSessionHandlerInterface
+    /**
+     * @desc This is for forward compatibility, the custom Session implementation is not being used anymore in Pimcore 11.
+     * @desc For forward compatibility, you can use this class and pass the SessionInterface from the request, in Pimcore 10.6, the Admin Session will be used instead
+     *
+     * @param SessionInterface $session Session being used is the Admin Session, you can safely pass nothing here
```

Comment:
How to pass "nothing" if it's not nullable? 

Apart from that: would it be forward compatible if "nothing" is passed?

Reply:
thanks, clarified it